rdf-delta-server-local/src/main/java/org/seaborne/delta/server/local/patchstores/zk/PatchStoreZk.java

```diff
@@ -135,18 +116,15 @@ private void updateLogChanges(Set<String> names) {
         Set<String> newLogs = SetUtils.difference(names, lastSeenLocal);
         Set<String> deletedLogs = SetUtils.difference(lastSeenLocal, names);
 
```

Comment:
I deleted the following because they were commented out and did not use the correct logging objects.